linux/devdoc/completion_port_linux_requirements.md

```diff
@@ -0,0 +1,187 @@
+# completion_port_linux requirements
+
+## Overview
+
+The completion_port_linux module handles the threading and control of the epoll socket system
+
+## Exposed API
+
+```C
```

Comment:
I didn't think there was a difference, and after a short bing search I didn't see anywhere it would say that I should use a lower case or an upper case.